tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaHelper.cs

```diff
@@ -94,6 +95,54 @@ internal static class KafkaHelper
             return scope;
         }
 
+        private static long TryGetSize(object obj)
+        {
+            if (obj == null)
+            {
+                return 0;
+            }
+
+            if (obj is byte[] bytes)
+            {
+                return bytes.Length;
+            }
+
+            if (obj is string str)
+            {
+                return Encoding.UTF8.GetByteCount(str);
+            }
+
+            return 0;
+        }
```

Comment:
nit: Just an FYI, but you can write this as a neat switch statement these days
```suggestion
        private static long TryGetSize(object obj)
            => obj switch
            {
                null => 0,
                byte[] bytes => bytes.Length,
                string str => Encoding.UTF8.GetByteCount(str),
                _ => 0;
             };
```

Reply:
Nice! Thank you for the suggestion